src/components/__tests__/addons.test.js

```diff
@@ -0,0 +1,95 @@
+import React from "react"
+import renderer from "react-test-renderer"
```

Comment:
Thanks for your efforts, looks good!
But we shall rely only on one testing library use `enzyme-to-json` library and do not use `react-test-renderer`  - there is no trace of it in `master` now. Check how to use it in other test files.

Reply:
Hi @Citrinin done